modules/sdk-core/src/bitgo/lightning/lightning.ts

```diff
@@ -114,4 +118,32 @@ export class Lightning implements ILightning {
       throw new Error(`error(s) parsing response body: ${errors}`);
     });
   }
+
+  public async decodeLnurlPay(lnurl: string): Promise<LnurlPayResponse & { domain: string }> {
+    const url = decodeLnurl(lnurl);
+    const body = await this.bitgo.get(url).result();
+    const decodedRes = decodeOrElse(LnurlPayResponse.name, LnurlPayResponse, body, (errors) => {
+      throw new Error(`error(s) parsing lnurl response: ${errors}`);
+    });
+    const { availableBalance } = await this.getBalance();
+    const domain = new URL(url).hostname;
+
+    decodedRes.maxSendable = Math.min(decodedRes.maxSendable / 1000, availableBalance);
+    decodedRes.minSendable = decodedRes.minSendable / 1000;
```

Comment:
Round this up for the same reasons as above, so `Math.ceil`

Note: This would mean we couldn't pay LNURL requests that request an exact millisatoshi amount. We'd either need to bite the bullet and start working with milllisats, or just accept that we won't be able to pay those (in practice I figure those will be super rare). Let's discuss a bit more tho.